src/backend/web/app.js

```diff
@@ -3,6 +3,11 @@ const express = require('express');
 const expressHandlebars = require('express-handlebars');
 const healthcheck = require('express-healthcheck');
 const cors = require('cors');
+const { ApolloServer } = require('apollo-server-express');
+
+// Required for GraphQl
+const storage = require('../utils/storage');
+const { typeDefs, resolvers } = require('./graphql/queries');
```

Comment:
If the only thing in the `./graphql` dir is this one file, let's rename it to `./graphql/index.js` so you can do `require('./graphql');` here.

Reply:
Changed to `index.js` and no we have `require('./graphql')`